gamemodes/base/entities/entities/lua_run.lua

```diff
@@ -6,21 +6,62 @@ ENT.DisableDuplicator = true
 
 AccessorFunc( ENT, "m_bDefaultCode", "DefaultCode" )
 
+local ALLOWED_LUA, ALLOWED_MAPS = include( "cfc_block_luarun/config.lua" )
+local CURRENT_MAP_ALLOWED = ALLOWED_MAPS[game.GetMap()]
+
 function ENT:Initialize()
+
+	-- If the entity has its first spawnflag set, run the code automatically
+	if self:HasSpawnFlags( SF_LUA_RUN_ON_SPAWN ) then
+		self:RunCode( self, self, self:GetDefaultCode() )
+	end
+
 end
 
 function ENT:KeyValue( key, value )
+
```

Comment:
nit pick, line 22 and 26 here don't really need the spaces, same for 13 and 18 and such.

Reply:
👍